autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/ExpandedCourseCard.css

```diff
@@ -26,6 +26,7 @@
     align-items: center;
     cursor: pointer;
     user-select: none;
+    margin: 3px;
 }
 
 .content {
```

Comment:
Change these properties in `.content` to make it so when a course has too many sections, it will overflow (I can't actually make suggestions because you didn't edit these lines):
```css
{
    padding: 8px;
    overflow: auto;
    max-height: 400px; /* could be more/less, see how you like different values of this */
}
```
Also remove the `margin` property so that the scrollbar will show on the right edge rather than 8px inside the card.

Reply:
Yeah this is a good point, we should definitely limit a course card's height as we don't want one course with a bunch of sections to take up a massive portion of the column. That being said, it might look weird having two scroll bars right next to each other. I'll try to implement it and report back

Reply:
So I tried this two different ways. One by adding the scrollbar to the entire card, then another by adding it on the `SectionSelect` component rather than the entire card. It doesn't happen as much with the `SectionSelect` one, but it's kind of annoying having two scroll bars since you basically have to hover wherever the course-card scroll area is _not_, which is a pretty small area when you have it in section select.
 
Top picture is what ryan suggested, bottom is it on the `SectionSelect`.
![chrome_RSWrLiS23h](https://user-images.githubusercontent.com/7295783/77988511-29578b80-72e2-11ea-9a85-6404f583634a.png)
![dPfBk8hWax](https://user-images.githubusercontent.com/7295783/77988515-2bb9e580-72e2-11ea-8a62-2be79d2189a7.png)

Reply:
Although putting it on the `SectionSelect`'s List does make it so that the first prof name won't scroll with the rest of the section rows, any idea what may be causing this @firejake308 ?

![chrome_XIlM6VIYGK](https://user-images.githubusercontent.com/7295783/77989003-493b7f00-72e3-11ea-90c2-d2ef1e9204fb.png)

Reply:
This is what it should look like if you make the changes I suggested:
![image](https://user-images.githubusercontent.com/28655462/78037358-0dc7a180-7331-11ea-97e7-19dd9f0a5a6b.png)
I think you forgot to remove the margin, this is what `.content` looks like:
```css
.content {
    padding: 8px;
    overflow: auto;
    max-height: 200px;
    align-self: stretch;
    display: flex;
    flex-direction: column;
    color: black;
}
```